docs/zkapps/o1js/keccak.mdx

```diff
@@ -77,7 +74,9 @@ bytes.bytes.forEach((b) => console.log(b.toBigInt()));
 // [100n, 111n, 103n, 0n, 0n, 0n, 0n, 0n, 0n, 0n, 0n, 0n, 0n, 0n, 0n, 0n]
 ```
 
-_Note:_ If the input array is smaller than the length of the `Bytes` class, it will be padded with zeros.
+:::note
```

Comment:
take a look at the preview and see if we need the Note style applied, it's very prominent

https://docs2-git-feature-ecdsa-keccak-overview-minadocs.vercel.app/zkapps/o1js/keccak#basic-usage